src/sentry/features/manager.py

```diff
@@ -112,8 +94,8 @@ def has(self, name, *args, **kwargs):
         # Features are by default disabled if no plugin or default enables them
         return False
 
-    def _get_plugin_value(self, feature, actor):
-        for handler in self.handlers:
+    def _get_handler(self, feature, actor):
+        for handler in self._handler_registry.get(feature.name, ()):
```

Comment:
Nice efficiency improvement here, great that removing this abstraction allowed us to refactor like this.